app/services/persona_detail_extraction.py

```
@@ -0,0 +1,70 @@
+            
+            return extracted_data
+        else:
+            logger.warning("LLM returned empty content for persona details extraction.")
+            return None
+
+    except json.JSONDecodeError as e:
+        logger.error(f"Failed to parse JSON from LLM response: {e}")
+        return None
+    except Exception as e:
+        logger.error(f"Error during LLM persona details extraction: {e}")
+        return None 
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)